package.json

```diff
@@ -12,12 +13,15 @@
     "dotenv": "^8.2.0",
     "graphql": "^15.5.0",
     "graphql-request": "^3.4.0",
+    "jquery": "^3.6.0",
     "node-sass": "^5.0.0",
+    "popper.js": "^1.16.1",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
+    "react-icons": "^4.2.0",
```

Comment:
do you know where these additional dependencies are from in your package.json

Reply:
No, I'm not sure...it could be from the installation process of multiple different things or issues the file paths which I am somewhat confused about